packages/feedback-service/package.json

```diff
@@ -23,7 +23,7 @@
     "dist"
   ],
   "scripts": {
-    "test": "jest --forceExit",
+    "test": "cross-env NODE_ENV=test DB_PATH=localhost DB_NAME=feedback-db-test jest --forceExit",
```

Comment:
I tried this having test.env  file inside e2e/test.env  and also on similar path of .env but was not able to read value from this file.
Do i need any other extra setting like webpack or something  ?

Reply:
The `.test.env` needs to be in the `src/e2e/` directory. Check the webpack config. When testing, it automatically injects the variables from `.test.env` into the environment.

Reply:
"build:dev": "webpack --config webpack.dev.js", === >module.exports = merge(webpack.common.js, { 
thats how its reaching up to comman webpack in case of unit testing command-webpack not being called i tried it.

Reply:
Actually you're right @anjneeksharma 

@riginoommen Why's the .test.env configured inside the webpack?
It needs to be configured in the test files.
Also, have you tested it on user-service? Was the test.env working there?

Reply:
Yes its working for me

Reply:
@riginoommen  when i run my npm test webpack.comman.js is not being called then how it will set envPath ?  
webpack.comman.js is being called from webpack.dev.js and and  webpack.prod.js and those prod and dev webpack will be called only when npm run start or npm run start:dev is being run not in case of npm test.

Reply:
@riginoommen  i think why its working for you because you execute 2 command together npm run start:dev && npm test if you will execute only one command npm test it will fail for you also if i am not wrong.

Reply:
@deshmukhmayur  i gone through multiple article and youtube video what i found 
Environment variable can be set in 3 place:-
  1 project.json
  2 jest.config.ts 
  3 inside test file itself in beforeAll() function 

we can keep our env variable in src/e2e/.test.env file as well but in that case we need to use npm test --env=<environment>  meaning   **npm test --env src/e2e** this will also work fine.

Please let me know which one i should use.